modules/kueue-monitoring-pending-workloads-on-demand.adoc

```diff
@@ -0,0 +1,99 @@
+// Module included in the following assemblies:
+//
+// * ai_workloads/kueue/monitoring-pending-workloads.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="monitoring-pending-workloads-on-demand_{context}"]
+= Monitoring pending workloads on demand
+
+[role=”_abstract”]
+To enable monitoring of pending workloads, you must configure ClusterQueue and LocalQueue assets, create a job manifest file that defines monitoring settings, and create a label for the default Kueue namespace. You can also test job monitoring.   
```

Comment:
it's not necessary to mention: "that defines monitoring settings"
it doesn't seem correct.